src/command.ts

```diff
@@ -52,18 +60,54 @@ export class Command<C extends Context = Context> extends Composer<C> {
     const scope: BotCommandScope | null = match({ type, chatId, userId })
       .with(
         { type: "default" },
+        ({ type }) => {
+          this.command(this.names, ...this.#middleware);
+          return { type };
+        },
+      )
+      .with(
         { type: "all_chat_administrators" },
+        ({ type }) => {
+          this
+            .filter(isAdmin)
```

Comment:
Isn't this going to perform many requests now in some cases?

Reply:
One per user AFAIK, and it's the only way to check for admins currently so either we go with it (maybe add some sort of cache) or we remove this functionality for this chat type, which leaves the API inconsistent